modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -66,3 +66,26 @@ Defaults to 4 seconds.
 The presence ping gather job parameter changes the timeout setting for gathering the presence ping, in seconds.
 Adjust the [systemitem]``java.salt_presence_ping_gather_job_timeout`` parameter.
 Defaults to 1 second.
+
+
+== Disabling the Salt Mine
+
+In older versions, {productname} used a tool called Salt mine to check minion availability.
```

Comment:
We could add how much load we generated before, so somehow we explain why the new way is better. Something like:
"In older versions, {productname} used a tool called Salt mine to check minion availability, pinging the server every hour."